common/djangoapps/student/models.py

```diff
@@ -1677,17 +1677,36 @@ def upgrade_deadline(self):
         Returns:
             datetime|None
         """
+        log.debug('Determining upgrade deadline for CourseEnrollment %d...', self.id)
```

Comment:
They are useful for local debugging. Since they are at the debug level, they won't be seen in production.